apps/cowswap-frontend/src/common/pure/NewModal/index.tsx

```diff
@@ -145,16 +151,22 @@ export interface NewModalProps {
   title?: string
   onDismiss?: () => void
   children?: React.ReactNode
+  screenMode?: boolean
```

Comment:
Use `modalMode` to match the prop from the other stuff?

Reply:
Fixed: https://github.com/cowprotocol/cowswap/pull/3756/commits/08afda61eca55c270ff0dac10aa8a68c5a88ef6d